### ISSUE TITLE
SMU code differences compared to Flygoat's version

### ISSUE THREAD

**Issue by Road-Drum** [irusanov/ZenStates#1]
From the readme, it was mentioned that `0x3` and `0x4` refers to EnableSMUFeatures and DisableSMUFeatures. However, from Flygoat's version it is a totally different command.  

I have tried Flygoat's version in my laptop, which works as intended. Is your SMU message codes are different because of SMU version and/or platform difference? The original Asus Zenstate 0.7.1 message codes are identical to Flygoat's version.  

I'd like to try yours on my ryzen laptop but I want to make sure that I didn't change anything harmful first. Maybe I can check my SMU version on my laptop first if I have time

**Comment by irusanov** [comment]
Yes, there are differences between platforms, CPU generations on the same platform and sometimes even between SMU versions.
However, this should be fixed for gen1.
I'm currently rewriting that part, so it will be easier to support different generations with their own SMU addresses.

**Comment by Road-Drum** [comment]
Thanks, I've tried your Zenstates ver. 0.7 and 0.8 and both of them causes Raven Ridge (FP5, not AM4) to enter 400MHz mode. Clicking any of the checkboxes causes it to behave like this so I assume some of the checkboxes are calling SMU messages that are supposed to be for AM4 platform.  

Also is the SMU message ID in README.md is for zen2 only? I currently have a theory that A320 board is actually OC disabled by SMU instead of the chipset which is probably the reason why some A320 suddenly capable of OC. OCEnable flag would be a good SMU command to send but I'd like to be sure which message is which first

**Comment by irusanov** [comment]
Yes, messages are different, I will soon add RavenRidge set of commands. Not all of them are supported.

**Comment by irusanov** [comment]
First version is up: https://github.com/irusanov/SMUDebugTool
You can set the addresses and command manually.
